website/docgen/dgeni-config.js

```diff
@@ -71,14 +71,14 @@ myPackage.config(function(readFilesProcessor, templateFinder, writeFilesProcesso
   readFilesProcessor.basePath = path.resolve(__dirname, '../..');
 
   readFilesProcessor.sourceFiles = [
-    {include: 'lib/**/element.js'},
-    {include: 'lib/**/protractor.js'},
-    {include: 'lib/**/locators.js'},
-    {include: 'lib/**/expectedConditions.js'},
-    {include: 'node_modules/selenium-webdriver/lib/**/locators.js'},
-    {include: 'node_modules/selenium-webdriver/lib/webdriver/webdriver.js'},
-    {include: 'node_modules/selenium-webdriver/lib/webdriver/key.js'},
-    {include: 'node_modules/selenium-webdriver/lib/webdriver/promise.js'}
+    {include: 'built/element.js'},
+    {include: 'built/protractor.js'},
+    {include: 'built/locators.js'},
+    {include: 'built/expectedConditions.js'},
+    {include: 'built/selenium-webdriver/locators.js'},
+    {include: 'built/selenium-webdriver/webdriver.js'}
+    // {include: 'node_modules/selenium-webdriver/lib/webdriver/key.js'},
```

Comment:
Don't we still want key and promise, for now? (Until we make a change to link to webdriver docs).